README.md

```diff
@@ -6,8 +6,7 @@ There are 4 rule sets:
 1. `eslint-config-digitalbazaar`: Base rules for both node and the browser.
 2. `eslint-config-digitalbazaar/jsdoc`: Rules for JSDoc for both node and the browser.
 3. `eslint-config-digitalbazaar/module`: Rules for modules for both node and the browser.
-4. `eslint-config-digitalbazaar/vue`: Rules for Vue 2 projects and browser only.
-5. `eslint-config-digitalbazaar/vue3`: Rules for Vue 3 projects and browser only.
```

Comment:
The last 3 commits look like they mistakenly reverted some vue3 and style changes here and elsewhere.

Reply:
Excellent catch I've decided to just close this PR and move the work to another pr here: https://github.com/digitalbazaar/eslint-config-digitalbazaar/pull/66

That PR does not modify the README. I believe the issue is rebasing the README with the vue changes and the cjs changes resulted in some weird git history.